x-pack/plugins/ml/public/components/full_time_range_selector/full_time_range_selector_service.ts

```diff
@@ -7,10 +7,10 @@
 import moment from 'moment';
 
 import { i18n } from '@kbn/i18n';
-import { Query } from 'ui/embeddable';
 import { IndexPattern } from 'ui/index_patterns';
 import { toastNotifications } from 'ui/notify';
 import { timefilter } from 'ui/timefilter';
+import { Query } from 'src/legacy/core_plugins/embeddable_api/public/index';
```

Comment:
@lukeelmers can we use absolute paths like this ?
never saw this used, should we change this to
my suggestion would be to change this to `plugins/embeddable_api` or to relative path (not sure about relative from x-pack either)

Reply:
Can't do relative - see https://github.com/elastic/kibana/issues/36900.  @mistic is working on a fix I believe.  `plugins/embeddable_api` didn't work either (at least not without extra setting modifications in tsconfig.json or somewhere else).

Reply:
Yeah there's something going on with the tsconfig I think, because this is only an issue in ts files -- the webpack alias works as expected in JS files.

This is the relevant tsconfig line. I was just running into this same thing today so I'm trying to experiment with a solution, because it's going to make things much easier to migrate later if we can just use the plugins/ alias.

https://github.com/elastic/kibana/blob/924b230ca411fa41e647c49e8ef019213430d7cb/tsconfig.json#L9